ang/civiawards/dashboard/directives/more-filters-dashboard-action-button.controller.js

```diff
@@ -91,7 +91,10 @@
         .then(processAwardTypeFilters)
         .then(function (awardTypeIds) {
           var param = {
-            case_type_id: awardTypeIds.length > 0 ? { IN: awardTypeIds } : ''
+            case_type_id: awardTypeIds.length > 0 ? { IN: awardTypeIds } : '',
```

Comment:
If we are already having the same check inside `processMyAwardsFilter`, do we need this filter anymore? Because `processMyAwardsFilter` will return only those Case Types which are Active(or not active), and we use those Case Type IDS to fetch further items in dashboard, so not sure why this is needed.

Reply:
@deb1990 this is needed because even if we send specific case type IDs, the dashboard overview will directive will request active case types:
https://github.com/compucorp/uk.co.compucorp.civicase/pull/402/files#diff-6576be8656588616fa32fd3f5e72c5b9R171

So this basically is forcing the case overview to display disabled ones.

Also, we need the case overview to force enabled case types because that's the default behaviour, otherwise it would show all cases.